pages/common/content/index.js

```diff
@@ -10,7 +10,8 @@ module.exports = {
       read: 'Details',
       update: 'Amend establishment details',
       apply: 'Apply for licence',
-      fees: 'Estimated licence fees'
+      fees: 'Estimated licence fees',
+      rops: 'Return of procedures'
```

Comment:
It's `Returns` (plural) below. Should it also be here?

Reply:
yes